assets/src/components/with-animation-controls.js

```diff
@@ -1,22 +1,91 @@
 /**
  * WordPress dependencies
  */
-import { createHigherOrderComponent } from '@wordpress/compose';
+import { createHigherOrderComponent, compose } from '@wordpress/compose';
 import { InspectorControls } from '@wordpress/editor';
 import { Fragment } from '@wordpress/element';
 import { PanelBody } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { withDispatch, withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { ALLOWED_BLOCKS } from '../constants';
-import { AnimationControls, withParentBlock } from './';
+import { AnimationControls } from './';
+
+const applyWithSelect = withSelect( ( select, props ) => {
+	const { getSelectedBlockClientId, getBlockRootClientId, getBlock, getBlocksByClientId, getBlockOrder } = select( 'core/editor' );
+	const { getAnimationOrder } = select( 'amp/story' );
+
+	const currentBlock = getSelectedBlockClientId();
+	const page = getBlockRootClientId( currentBlock );
+
+	const { ampAnimationAfter } = props.attributes;
+	const predecessor = getBlocksByClientId( getBlockOrder( page ) ).find( ( b ) => b.attributes.anchor === ampAnimationAfter );
+
+	return {
+		parentBlock: getBlock( getBlockRootClientId( props.clientId ) ),
+		animationAfter: predecessor ? predecessor.clientId : undefined,
+		getAnimatedBlocks() {
+			const { getBlockType } = select( 'core/blocks' );
+
+			const animatedBlocks = getAnimationOrder()[ page ] || [];
```

Comment:
There's an error in the console that seems to refer to this line, is this happening for you as well? (Note: The line numbers in the console don't match the ones in the source.)
![screen shot 2019-03-01 at 4 01 49 pm](https://user-images.githubusercontent.com/3294597/53646539-b6419880-3c3b-11e9-8af9-1f859b87ce5f.png)

Reply:
Additional information: The error occurs when creating a few blocks, assigning animations and then refreshing the page.

Reply:
Looks like there is more going on here, sometimes the content just disappears now. Ugh.

Reply:
Should be fixed now with ebd26139cd7f2463803c2992f1149a45f354d02b.